File: array/duplicate.swift
import Foundation

func containsDuplicate(_ nums: [Int]) -> Bool {
    let sortedNums = nums.sorted()
    let count = sortedNums.count

    for j in 0..<count - 1 {
        if sortedNums[j] == sortedNums[j + 1] {
            return true
        }
    }
    return false
}

let nums = [1, 2, 3, 1]
print(containsDuplicate(nums))

File: array/duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
  nums.sort((a, b) => a - b);
  for (let j = 0; j < nums.length - 1; j++) {
    if (nums[j] === nums[j + 1]) {
      return true;
    }
  }
};

console.log(containsDuplicate([1, 2, 3, 4, 5, 6, 7, 8, 9, 6]));

module.exports = {
  containsDuplicate,
};
